admin/resources/js/components/UserTable.tsx

```diff
@@ -47,9 +47,22 @@ const UserTable: React.FC<UserTableProps> = (users) => {
 
   const data = useMemo(() => users, [users]);
 
+  const {
+    getTableProps,
+    getTableBodyProps,
+    headerGroups,
+    rows,
+    prepareRow,
+    state,
+    visibleColumns,
+    preGlobalFilteredRows,
+    setGlobalFilter,
+  } = useTable({ columns, data: data.users }, useGlobalFilter, useSortBy);
+
   return (
     <>
-      <Table data={data} columns={columns} />
+      <p> Waffles </p>
```

Comment:
Waffles are delicious! I approve of their consumption but not of their inclusion between paragraph tags!

Reply:
Ha. Wops. I removed it but forgot to push it. My mistake!